flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/table/JdbcOutputFormatBuilder.java

```diff
@@ -100,10 +98,7 @@ public JdbcOutputFormatBuilder setFieldDataTypes(DataType[] fieldDataTypes) {
             return new JdbcOutputFormat<>(
                     new SimpleJdbcConnectionProvider(jdbcOptions),
                     executionOptions,
-                    ctx ->
-                            createBufferReduceExecutor(
-                                    dmlOptions, ctx, rowDataTypeInformation, logicalTypes),
```

Comment:
since all the usages of `rowDataTypeInformation` are removed, could we remove the field itself?

Reply:
Done